napalm/ios/ios.py

```diff
@@ -1955,6 +1955,54 @@ def get_snmp_information(self):
             snmp_dict['chassis_id'] = snmp_chassis
         return snmp_dict
 
+    def get_users(self):
+        """
+        Returns a dictionary with the configured users.
+        The keys of the main dictionary represents the username.
+        The values represent the details of the user,
+        represented by the following keys:
+
+            * level (int)
+            * password (str)
+            * sshkeys (list)
+
+        *Note: sshkeys on ios is the ssh key fingerprint
+
+        The level is an integer between 0 and 15, where 0 is the
+        lowest access and 15 represents full access to the device.
+        """
+        users = {}
+        command = "show run | section username"
+        output = self._send_command(command)
+        curuser = ""
+        for line in output.splitlines():
+            if line.startswith("username"):
+                tokens = line.split(" ")
```

Comment:
Why not just `line.split()` here it will split on consecutive white space...so probably marginally less fragile.